src/codesyncd/websocket/socket_events.ts

```diff
@@ -143,8 +184,29 @@ export class SocketEvents {
                     default:
                         return false;
                 }
+            case EVENT_TYPES.TAB_PROCESSED:
+                switch (resp.status) {
+                    case HttpStatusCodes.OK:
+                        this.onTabProcessed(resp.tab_path);
+                        return true;
+                    case HttpStatusCodes.PAYMENT_REQUIRED:
+                        this.onPaymentRequiredRecieved();
+                        return true;
+                    case HttpStatusCodes.INVALID_USAGE:
+                        this.onTabProcessed(resp.tab_path);
+                        return true;
+                    case HttpStatusCodes.FORBIDDEN:
+                        this.onTabProcessed(resp.tab_path);
+                        return true;
+                    default:
+                        return false;
+                }
             default:
                 return false; 
         }
     }
 }
+function sendTabsToServer(connection: any, validTabs: ITabYML[]) {
```

Comment:
Didn't get this, why have we defined this?

Reply:
This must have been auto-generated, removing.